lib/core/Makefile

```diff
@@ -4,27 +4,48 @@ ifeq ($(UNAME), Darwin)
 	CLANG_PREFIX += AR=$(shell brew --prefix llvm)/bin/llvm-ar CC=$(shell brew --prefix llvm)/bin/clang
 endif
 
+LND_MACAROON_HEX=$(shell xxd -p ../../regtest/boltz/data/lnd1/data/chain/bitcoin/regtest/admin.macaroon | tr -d '\n')
```

Comment:
What is the behaviour here if the submodule does not exist? Should we check that before running the tests?

Reply:
If the submodule doesn't exist it won't be possible to start the regtest environment. I've added a check so that we get an immediate and helpful error message if the regtest setup isn't running. [8be6360](https://github.com/breez/breez-sdk-liquid/pull/816/commits/8be6360b4f764dcff49db3171d44a8cbc2baeda1)